packages/services/src/services/Locale/Locale.js

```diff
@@ -85,6 +85,76 @@ const _axiosConfig = {
  */
 const _sessionListKey = 'dds-countrylist';
 
+/**
+ * Use the <html> lang attr to determine a return locale object
+ *
+ * @returns {object}
+ * @private
+ */
+const _getLocaleByLangAttr = () => {
+  if (root.document.documentElement.lang) {
+    const lang = root.document.documentElement.lang.toLowerCase();
+    if (lang.indexOf('-') === -1) {
+      return _localeDefault;
+    } else {
+      const codes = lang.split('-');
+      return { cc: codes[1], lc: codes[0] };
+    }
+  } else {
+    return _localeDefault;
+  }
+};
+
+/**
+ * Return a locale object based on the DDO API, or "false"
+ * so the consumer can decide what to do next
+ *
+ * @returns {(object | boolean)}
+ * @private
+ */
+async function _getLocaleFromDDO() {
+  const ddoLocal = await DDOAPI.getAll();
+
+  const lang = root.document.documentElement.lang || undefined;
+  let pageInfoIBM = ddoLocal.page.pageInfo.ibm;
```

Comment:
Let's move this inside the if block right below in case the ddo object is malformed or nonexistent.